test/plugins/features/movies/index.test.js

```diff
@@ -20,4 +26,80 @@ describe('movies integration', () => {
 
   });
 
+  describe('list', () => {
+
+    it('Lists all movies', () => {
+      return Movies.inject({
+        url: '/movies',
+        method: 'GET'
+      })
+      .then((response) => {
+        expect(response.statusCode).to.eql(200);
+      });
+    });
+
+    it('Lists movies for a given release year', () => {
+      return Movies.inject({
+        url: '/movies?year=2011',
+        method: 'GET'
+      })
+      .then((response) => {
+        expect(response.statusCode).to.eql(200);
+        expect(response.result.length).to.be.eq(1);
+        expect(response.result[0]).to.have.all.keys([
+          'id',
+          'title',
+          'release_year',
+          'object'
+        ]);
+      });
+    });
+
+    it('Lists movies for a given range of release years', () => {
```

Comment:
Nit: Its not a hard and fast rule but normally I'd expect tests that check most of the functionality (different filter methods) of the controller to be in the controller unit tests and for integration tests to handle more of the end-to-end testing of the endpoint (validation, authentication, error codes).